# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../koffice/krita-uic-issue.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur koffice-1.4.2.orig/krita/plugins/tool_star/wdg_tool_star.ui koffice-1.4.2/krita/plugins/tool_star/wdg_tool_star.ui
--- koffice-1.4.2.orig/krita/plugins/tool_star/wdg_tool_star.ui	2005-10-05 13:27:00.000000000 +0300
+++ koffice-1.4.2/krita/plugins/tool_star/wdg_tool_star.ui	2005-10-11 11:19:54.000000000 +0300
@@ -98,6 +98,11 @@
         </spacer>
     </grid>
 </widget>
+<customwidgets>
+    <customwidget>
+	<header location="global">knuminput.h</header>
+    </customwidget>
+</customwidgets>
 <tabstops>
     <tabstop>verticesSpinBox</tabstop>
     <tabstop>ratioSpinBox</tabstop>
diff -Nur koffice-1.4.2.orig/krita/ui/wdgmatrix.ui koffice-1.4.2/krita/ui/wdgmatrix.ui
--- koffice-1.4.2.orig/krita/ui/wdgmatrix.ui	2005-10-05 13:27:26.000000000 +0300
+++ koffice-1.4.2/krita/ui/wdgmatrix.ui	2005-10-11 10:59:01.000000000 +0300
@@ -338,5 +338,10 @@
         </widget>
     </grid>
 </widget>
+<customwidgets>
+    <customwidget>
+	<header location="global">knuminput.h</header>
+    </customwidget>
+</customwidgets>
 <layoutdefaults spacing="6" margin="11"/>
 </UI>
